dependencies/src/test/java/com/google/cloud/tools/opensource/dependencies/DependencyGraphIntegrationTest.java

```diff
@@ -101,10 +105,10 @@ public void testJaxen()
     DependencyGraph graph = DependencyGraphBuilder.getCompleteDependencies(jaxen);
     
     List<Update> updates = graph.findUpdates();
-    Truth.assertThat(updates).hasSize(5);
+    Truth.assertThat(updates).hasSize(8);
```

Comment:
Updates to these tests are due to the changes to resolveCompileTimeDependencies to include provided scope.